dask/highlevelgraph.py

```diff
@@ -1271,13 +1320,52 @@ def to_graphviz(
         attrs.setdefault("fontsize", str(node_size))
         attrs.setdefault("tooltip", str(node_tooltips))
 
+        if color == "layer_type":
+            layer_colors = {
```

Comment:
Possible future improvement: it feels like these should be (class) attributes of the layers themselves. Since colouring is not going to be default yet, it's ok to leave them here for now.